app/services/concerns/xccdf/profiles.rb

```diff
@@ -30,6 +31,14 @@ def old_profiles
           ref_id: @op_profiles.map(&:id), benchmark: @benchmark&.id
         ).index_by(&:ref_id)
       end
+
+      def value_overrides(op_profile)
+        @cached_value_overrides ||= @op_value_definitions.index_by(&:id)
```

Comment:
You already have this cached if you look at `value_definition_for` (omitted the ID as I explained below). No need to cache it twice...

Reply:
This cache is different though because we're indexing the `OpenscapParser::Value`s instead of `ValueDefinition`s

Reply:
Isn't it the same after importing?

Reply:
The key is to import the definitions first :wink: